README.rdoc

```diff
@@ -46,6 +46,19 @@ circle.yml:
             files:
               - test/**/*.rb
 
+Alternatively, you can add the following code to your test runner setup:
+
+    # test/helper.rb
+    require 'minitest'
+    require 'minitest/autorun'
+    require 'minitest/ci'
+
+    Minitest::Ci.report_dir = "#{ENV["CIRCLE_TEST_REPORTS"]}/reports"
```

Comment:
Good point, something like that should work.

Hmm do we set `$CI` or `$CIRCLECI` or similar?